recipes/iDQ/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "iDQ" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "31efed79f9a0e0eb03ef174f5f8e48d3d23a3213dd95616a89730661fb385669" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k or win]
```

It looks like you removed no arch because you need Python 2? If so you can use the `python <3` syntax instead and keep this as a noarch package.
```suggestion
  noarch: python
```